packages/client/src/utils/index.ts

```diff
@@ -126,18 +126,19 @@ export async function getVercelIgnore(
   prebuilt?: boolean,
   rootDirectory?: string
 ): Promise<{ ig: Ignore; ignores: string[] }> {
-  let ignores: string[] = [];
-
-  const outputDir = posix.join(rootDirectory || '', '.output');
+  const ig = ignore();
+  let ignores: string[];
 
   if (prebuilt) {
-    ignores.push('*');
+    const outputDir = posix.join(rootDirectory || '', '.vercel/output');
```

Comment:
```suggestion
    const outputDir = posix.join(rootDirectory || '', '.vercel', 'output');
```

Reply:
Same thing really 🤷

Reply:
Now that I think about it, the `outputDir.split('/')` on line 135 wont work on windows

Reply:
It's using `posix.join()`